doc fixes for jwt location (#544)
agentgateway/website: content/docs/standalone/main/configuration/security/backend-authn.md

```yaml
backendAuth:
  key:
    value: $MY_API_KEY
```

A file path can also be used:

```yaml
backendAuth:
  key:
    value:
      file: /path/to/my/key
```

> By default, the key is sent as the `Authorization` header value. Use the `location` field to change where the key is sent:

Reviewer: ```suggestion
By default, the proxy retrieves the key from the `Authorization` header value. To use a different header name, use the `location` field as shown in the following example:
```

```yaml
backendAuth:
  key:
    value: $MY_API_KEY
    location:
      # Send as a request header (default)
      header:
        name: authorization
        prefix: "Bearer "
```

```yaml
backendAuth:
  key:
    value: $MY_API_KEY
    location:
      # Send as a query parameter
      queryParameter:
        name: api_key
```

```yaml
backendAuth:
  key:
    value: $MY_API_KEY
    location:
      # Send as a cookie
      cookie:
        name: api_key
```

> When using any form of incoming authentication (such as [JWT]({{< link-hextra path="/configuration/security/jwt-authn/" >}}), [API key]({{< link-hextra path="/configuration/security/apikey-authn/" >}}), or [basic auth]({{< link-hextra path="/configuration/security/basic-authn/" >}})), the original credential is removed from the request by default before forwarding to the backend.

Reviewer: ```suggestion
When using any form of incoming authentication, such as [JWT]({{< link-hextra path="/configuration/security/jwt-authn/" >}}), [API key]({{< link-hextra path="/configuration/security/apikey-authn/" >}}), or [basic auth]({{< link-hextra path="/configuration/security/basic-authn/" >}}), the original credential is removed from the request by default before forwarding to the backend.
```

> To pass the original credential through to the backend unchanged, use the `passthrough` method:

Reviewer: ```suggestion
To pass the original credential through to the backend, use the `passthrough` method:
```
